tools/lsp/language.rs

```diff
@@ -720,9 +695,6 @@ pub async fn reload_document(
         )?;
     }
 
-    // Tell Preview about the Components:
-    report_known_components(document_cache, ctx);
```

Comment:
i'm concerned that the preview would only know the component that are known to the preview. While the LSP should know more components from the whole project, and we may want to include these components even if they are not yet imported